app/src/main/java/io/gnosis/safe/ui/transactions/TransactionListViewModel.kt

```diff
@@ -121,9 +121,8 @@ class TransactionListViewModel
                 // that was submitted for execution
                 if (txLocal?.safeTxNonce == txListEntry.transaction.executionInfo?.nonce) {
                     // use submittedAt timestamp to distinguish between conflicting transactions
-                    if (txLocal?.submittedAt == txListEntry.transaction.timestamp.time && txListEntry.transaction.txStatus == TransactionStatus.AWAITING_EXECUTION) {
+                    if ((txLocal?.submittedAt == null || txLocal?.submittedAt == txListEntry.transaction.timestamp.time) && txListEntry.transaction.txStatus == TransactionStatus.AWAITING_EXECUTION) {
```

Comment:
I would add a comment explaining that submittedAt is null for incoming transfers